components/tikv_util/src/future.rs

```diff
@@ -40,14 +39,11 @@ where
 
 pub fn paired_must_called_std_future_callback<T>(
```

Comment:
Now this mod does not provide any util function that returns a 0.1 future, the `std` in the function name is useless now.